src/main/java/com/annapolisworks/OpNode.java

```diff
@@ -0,0 +1,27 @@
+package com.annapolisworks;
+
+public class OpNode {
+
+    public OpNode lastNode;
+    public OpNode nextNode;
+    public Operator operator;
+    public double lastNumber;
+    public double nextNumber;
```

Comment:
It's odd that all these are public. Normally you'd only see something like this on an inner class.